frontend/src/components/Profile/ProfilePreview/ProfileMeta/RecruiterProfileMeta.tsx

```diff
@@ -17,6 +19,7 @@ export const RecruiterProfileMeta = React.memo<Props>((props) => {
           className={cn(styles.recruiterMetaItem, typography.smallHeading)}
           key={item.name}
         >
+          {item.name === RecruiterProfileMetaItems.City && <IconLocation />}
```

Comment:
what if we need an icon for another item? can we somehow modify the incoming array?